_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+- **Accelerate** development and deployment cycles;
+- Increase software **transparency**;
+- Improve the **quality and security** of code with automated testing and through external contributions (by working in the open);
+- Attract IT **talent**;
+- Reduce vendor **lock-in** by increasing capacity to support source code; and
+- Obtain metrics/statistics for **reporting** and continuous improvement.
+
+The Architecture Review Committee (ARC) already endorsed the usage of 3 version control systems in [July 2019 (internal presentation)](https://dialogue/proj/ITCD-RMTI/SAS-SAS/Documents/Services/Projects/GC%20Code/ARC%20Presentation%20on%20GCCode%20Governance.pptx):
+
+- [GitHub](https://github.com): Share with public (Unclassified);
+- [GCcode (internal)](https://gccode.ssc-spc.gc.ca): Share with GC (up to Protected A); and
+- [Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/): Share within ESDC (up to Protected B).
+
+However, work remains in some areas to drive adoption.
+For instance, GCcode is not officially supported by SSC and lack some modern DevOps features (such as deployment to external cloud providers).
+This strategy’s goal is to define what the target state of the version control systems should be, and provide a roadmap in getting to this target state.
+
+This strategy complements existing IT initiatives, such as [ESDC IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), and supports them with new activities (see [Coherent set of actions](#coherent-set-of-actions)).
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) regarding usage of the Version Control Systems.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see [Coherent set of actions](#coherent-set-of-actions)).
+
+This policy becomes active when IT Products are to be built.
+Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
+
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+- ESDC Information Management Policy (being drafted)
+- ESDC Security Policy (being drafted)
+
+### Development and Operations Teams (incl. SSC) <!-- omit in toc -->
+
+1. Use a Git-based Version Control System (VCS) among the following options, in that order of preference:
```

Comment:
As it's a policy, should there be a pointer to the list of approved VCS instead of listing them here? (just thinking of some laws that still ask for faxes to be used :wink:

Reply:
That's a good point. However, I'm not aware of an existing list with preferences. I assume a good place to create one would be the OSS Mgmt Framework, so the related action "Update ESDC OSS Management Framework" would cover it.

Reply:
IMHO, "1. Use a Git-based Version Control System (VCS) ..." should read "1. Use a DevOps platform ..." - none of the platforms listed below (GitHub, GCCode (a rebranded GitLab), Azure DevOps) is a VCS, most of them are leveraging Git as a VCS but Azure DevOps can also leverage TFS (https://docs.microsoft.com/en-us/azure/devops/user-guide/source-control?view=azure-devops)...

Reply:
Good point, thanks! I specified DevOps platform :)

Reply:
@tsanov I agree with the differentiation between VCS and Coding platform: GitHub, GitLab, etc. are not Git but social coding platform built around Git.

I would also argue that TFVC should be gradually be replaced by Git as the default VCS to ensure the intended outcome: Decentralized VCS with DevOps capabilities and the ability to work in the open, which is part of the rules we have to align with. Not that we are all there yet but it should still be the desired end-state given that this is a strategy.

See this MS article for more details: https://docs.microsoft.com/en-us/azure/devops/repos/git/import-from-tfvc?view=azure-devops

Reply:
Also pointing out here that Azure DevOps now recommends using Git as the default VCS:
https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops